### PR TITLE
10-7959f-1 Removing Prefill

### PR DESCRIPTION
**Pull request by reiting**
## Summary
This PR is trying to completely remove prefill from form 10-7959f-1.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/89837

## Testing done

- [ ] *New code is covered by unit tests*
As this was removing code, there was no extra test to write.

## What areas of the site does it impact?
this form only

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback
NA

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -1725,9 +1721,8 @@ avs:
   api_jwt: abcd1234abcd1234abcd1234abcd1234abcd1234
 
 brd:
-  api_key: 'fake_key'
```
Comment (balexandr) [inline config/settings.yml:1728]: @reiting Change these back to single quotes as it's an unnecessary update for this PR.

File: app/models/form_profile.rb [hunk 2]
```diff
@@ -102,7 +102,7 @@ class FormProfile
     coe: ['26-1880'],
     adapted_housing: ['26-4555'],
     intent_to_file: ['21-0966'],
-    ivc_champva: %w[10-7959F-1 10-7959C],
+    ivc_champva: %w[10-7959C],
```
Comment (balexandr) [inline app/models/form_profile.rb:105]: Change this to `['10-7959C']`

File: config/settings.yml [hunk 3]
```diff
@@ -422,9 +422,6 @@ adapted_housing:
 intent_to_file:
   prefill: true
 
-ivc_champva:
-  prefill: true
-
```
Comment (balexandr) [inline config/settings.yml:427]: I think we still need this for prefill to work in general

Comment (reiting) [reply config/settings.yml]: So just leave this file as it was originally basically? Or should I remove 7959f-1 from line 1347?

File: app/models/form_profile.rb [hunk 4]
```diff
@@ -102,7 +102,7 @@ class FormProfile
     coe: ['26-1880'],
     adapted_housing: ['26-4555'],
     intent_to_file: ['21-0966'],
-    ivc_champva: %w[10-7959F-1 10-7959C],
+    ivc_champva: %w['10-7959C'],
```
Comment (balexandr) [inline app/models/form_profile.rb:105]: You have to remove the `%w` thats only for Arrays with multiple strings.

File: config/settings.yml (hunk 3 above)
Comment (balexandr) [reply config/settings.yml]: Yeah I don't think you need to do anything in settings. I don't see "7959f-1" anywhere in settings.yml let alone line 1347

File: app/models/form_profile.rb (hunk 4 above)
Comment (reiting) [reply app/models/form_profile.rb]: Ahhhh my bad!

File: config/settings.yml (hunk 3 above)
Comment (reiting) [reply config/settings.yml]: Ok cool!

**Review by balexandr — APPROVED** [review]

**Review by RachalCassity — APPROVED** [review]

**Review by cloudmagic80 — APPROVED** [review]

**Pull request merged**